### ISSUE TITLE
Некорректно выводится версия SNAPSHOT в команде `opm list`

### ISSUE THREAD

**Issue by ovcharenko-di** [EvilBeaver/os-hub-frontend#34]
Воспроизведение ошибочного поведения:

- установить SNAPSHOT-версию пакета, например `opm install vanessa-automation-single@SNAPSHOT`
- убедиться, что вывод команды `opm list` не содержит информации о том, что версия пакета `vanessa-automation-single` - `SNAPSHOT`

**Comment by ovcharenko-di** [comment]
@nixel2007

**Comment by ovcharenko-di** [comment]
@nixel2007 мне твое сообщение в чате и показалось странным, но я посмотрел последние коммиты, нашел что-то похожее тут и поверил на слово 😄 

скорее всего произошло следующее: пакет был запушен в dev канал, отчего в хабе создалась версия SNAPSHOT, но внутри самого пакета в файле `opm-metadata.xml` указана версия `<version>1.2.038</version>`

**Comment by nixel2007** [comment]
Да, все так и есть. Эти метаданные создаются, насколько я помню, опмом при сборке пакета. Соответственно править нужно на той стороне.

**Comment by ovcharenko-di** [comment]
@nixel2007 ну так opm сейчас тоже не при чем, получается. Когда выполняешь `opm build`, он берет данные для opm-metadata.xml из packagedef. Какая там версия указана, такая и будет записана в opm-metadata.xml.

А вот когда делаешь push в dev-канал, вот тогда уже os-hub-frontend на своей стороне записывает у себя эту версию как SNAPSHOT.

Мне видится, что в opm build должен быть добавлен параметр, например, `opm build --snapshot`, а os-hub-frontend должен публиковать пакет как есть.

**Comment by nixel2007** [comment]
то есть по сути переделать работу с push --dev на build --dev, да?

**Comment by EvilBeaver** [comment]
Может @ovcharenko-di поможет напилить?
